contracts/ValidatorSet.template

```diff
@@ -314,7 +313,11 @@ contract ValidatorSet is IValidatorSet, System, IParamSubscriber {
            documentation for more details
 */
   function felony(address validator, uint256 felonyRound, uint256 felonyDeposit)
-            external override onlySlash onlyIfCurrentValidator(validator) {
+            external override onlySlash {
+    if (currentValidatorSetMap[validator] == 0) {
```

Comment:
apparently the expected behavior in case of non-validator is to return rather than revert